.github/workflows/rspec.yml

```diff
@@ -0,0 +1,26 @@
+name: RSpec
+on: 
+  pull_request:
+    branches:
+      - master
+  push:
+    branches:
+      - master
+jobs:
+  tests:
+    name: Run RSpec tests
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ruby-version: ['2.6', '2.7']
+        # ruby-version: ['2.6', '2.7', '3.0', '3.1', 'head']
```

Comment:
Can this line be removed?

Reply:
It can but this is what line 15 wants to look like when we validate against Ruby 3.0+. At the moment these will fail.

I'll trying using the `continue-on-error` to allow these to run but not cause the workflow to fail.